config/initializers/extra_paths.rb

```diff
@@ -0,0 +1,6 @@
+#
+# Add some extra paths here
+Rails.application.configure do
+  config.paths.add File.join('app', 'api'), glob: File.join('**', '*.rb')
+  config.autoload_paths += Dir[Rails.root.join('app', 'api', '*')]
```

Comment:
)

2016-04-10 14:04 GMT+03:00 Sergey Hanchar <notifications@github.com>:

> In config/initializers/extra_paths.rb
> <https://github.com/DevilOrg/GetCargo/pull/2#discussion_r59131773>:
>
> > @@ -0,0 +1,6 @@
> > +#
> > +# Add some extra paths here
> > +Rails.application.configure do
> > +  config.paths.add File.join('app', 'api'), glob: File.join('**', '*.rb')
> > +  config.autoload_paths += Dir[Rails.root.join('app', 'api', '*')]
>
> I was checking it and yes, you was right ;)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/DevilOrg/GetCargo/pull/2/files/291d7cb0705a632b8b149dba4cd29ab4b6dfc36e#r59131773>
>